packages/components/styles/src/button/button.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
I dont understand... why is the styles folder duplicated under the components folder?